app/src/main/java/com/example/iou/activities/SplashScreenActivity.java

```diff
@@ -0,0 +1,29 @@
+package com.example.iou.activities;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+
+import com.example.iou.R;
+
+public class SplashScreenActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_splash_screen);
+        new Handler().postDelayed(new Runnable() {
+            @Override
+            public void run() {
+                //This method will be executed once the timer is over
+                // Start your app main activity
+                Intent i = new Intent(SplashScreenActivity.this, LoginActivity.class);
+                startActivity(i);
+                // close this activity
+                finish();
+            }
+        }, 1000);
```

Comment:
do you have any plans to use this activity to initialize any dependencies that could potentially take a long time?

Reply:
Not at the moment - I don't think I have dependencies that could take a long time right now.